wazimap_ng/points/admin/forms/category_admin_form.py

```diff
@@ -22,6 +27,23 @@ def __init__(self, *args, **kwargs):
                 self.fields["description"].initial = metadata.description
                 self.fields["licence"].initial = metadata.licence
 
+    def clean(self):
+        cleaned_data = super(CategoryAdminForm, self).clean()
+        document = cleaned_data.get('import_collection', None)
+        required_headers = ["name", "longitude", "latitude", "id", "status", "status last updated", "class", "volume",
```

Comment:
Hey @emre2038 
There are only 3 headers that are required : name, longitude & latitude